analyses/snv-callers/compare_snv_callers_plots.Rmd

```diff
@@ -157,27 +227,49 @@ callers_per_mutation %>%
 
 ```{r}
 # Save this plot
-ggplot2::ggsave(file.path(consensus_plots_dir, "upset_median_vaf_plot.png"))
+ggplot2::ggsave(file.path(plots_dir, "upset_median_vaf_plot.png"))
 ```
 
 ## Base Changes 
 
-Set up the summarized form of the base change information. 
+Summarize base change information per caller. 
 
 ```{r}
 # Summarize by the number of times each base change shows up in each category.
-perc_change_df <- vaf_df %>%
+perc_change_df <- all_caller %>% 
+  dplyr::mutate(base_change = paste0(Allele, ">", Reference_Allele)) %>%
+  # Case when only works for df not sql, so we will convert now
+  as.data.frame() %>%
```

Comment:
If we are pulling all_caller into memory ever, then we should probably do it up at the top (the first time we create it), and use the in-memory functions  for subsequent steps. If we select only the columns we will need for this analysis, it might not even be too big.

The key is we don't want to have to execute the big joins more than once, if we can avoid it.

Reply:
Ah. Good point. Will change.